xero-payroll-au.yaml

```
@@ -4291,6 +4582,151 @@ components:
+        reason:
+          description: The reason the error occurred
+          type: string
+          example: The property isSubjectToSuper is required.
+    EarningsRateV2:
+      type: object
+      description: Earnings Rate for https://api.xero.com/payroll.xro/2.0 Xero Payroll AU API (v2)
+      properties:
+        EarningsRateID:
+          description: Xero unique identifier for an earnings rate
+          type: string
+          format: uuid
+          example: e0eb6747-7c17-4075-b804-989f8d4e5d39
+        Name:
+          description: Name of the earnings rate (max length = 100)
+          type: string
+          example: PTO
+          maxLength: 100
+        EarningsType:
+          description: Indicates how an employee will be paid when taking this type of earning
+          $ref: '#/components/schemas/EarningsType'
+        RateType:
+          description: Indicates the type of the earnings rate
+          $ref: '#/components/schemas/RateType'
+        TypeOfUnits:
+          description: Type of units used to record earnings (max length = 50). Only When RateType is RATEPERUNIT
+          type: string
+          example: Fixed
+          maxLength: 50
+        CurrentRecord:
+          description: Indicates whether an earnings type is active
+          type: boolean
+          example: true
+        ExpenseAccountID:
+          description:  The account that will be used for the earnings rate
+          type: string
+          format: uuid
+          example: 2c01577b-d2d7-42b6-a7a3-1a38f5c8c454
+        RatePerUnit:
+          description: Default rate per unit (optional). Only applicable if RateType is RATEPERUNIT.
+          type: string
+          example: 10
+        MultipleOfOrdinaryEarningsRate:
+          description: This is the multiplier used to calculate the rate per unit, based on the employee’s ordinary earnings rate. For example, for time and a half enter 1.5. Only applicable if RateType is MULTIPLEOFORDINARYEARNINGSRATE
+          type: number
+          format: double
+          x-is-money: true
+          example: 1.5
+        FixedAmount:
+          description: Optional Fixed Rate Amount. Applicable only if RateType is FixedAmount
+          type: number
+          format: double
+          x-is-money: true
+          example: 50.30
+        IsSubjectToTax:
+          description: Payments of this type are subject to PAYG withholding
+          type: boolean
+          example: false
+        IsSubjectToSuper:
+          description: Payments of this type are subject to Superannuation Guarantee Contribution
+          type: boolean
+          example: false
+        IsReportableAsW1:
+          description: Whether the earnings rate is reportable or exempt from W1
+          type: boolean
+          example: false
+        AccrueLeave:
+          description:  Indicates that this earnings rate should affect accruing of leave. Only applicable if RateType is MULTIPLEOFORDINARYEARNINGSRATE
```

```suggestion
          description:  Indicates that this earnings rate should affect accruing of leave. 
```

Not sure if we need that extra part here, this feels like more just a generic description and the `Required if RateType is MultipleOfOrdinaryEarningsRate` is more for the POST only